eth_worker/_docker_worker_script.sh

```diff
@@ -20,7 +21,7 @@ else
   if [ "$ret" -ne 0 ]; then
     exit $ret
   fi
-  celery -A eth_manager worker --loglevel=INFO --concurrency=10 --pool=eventlet -Q=celery,high-priority
+  celery -A eth_manager worker --loglevel=INFO --concurrency=10 --pool=eventlet -Q=low-priority,celery,high-priority
```

Comment:
@enjeyw Was the blocking you mentioned happening with this flag set, or before?

Reply:
oops - i think this was a mistaken change - it supposed to be default and high-priority only! (otherwise low-priority clogs the default queue)